src/main/java/com/wojciechbarwinski/demo/epic_board_games_shop/OrderCronJob.java

```diff
@@ -21,7 +22,11 @@ public class OrderCronJob {
     @Value("${orders.cleanup.not-payed.timer-minutes}")
     private int notPayedTimer;
 
+    @Value("${orders.cleanup.send-payed.timer-minutes}")
```

Comment:
nie ma takiego słówka jak "payed" - jest "paid" - to ta wredna "2 forma czasownika" :D